examples/vad_transcriber/wavTranscription.md

```diff
@@ -5,7 +5,36 @@ They take in a wav file of any duration, use the WebRTC Voice Activity Detector
 to split it into smaller chunks and finally save a consolidated transcript.
 
 ### 0. Prerequisites
-Setup your environment
+#### 0.1 Install requiered packages
+Install the package which contains rec on the machine:
+
+Fedora:
+
+``` sudo dnf install sox ```
+
+Tested on: 29
+
+Ubuntu/Debian
+
+``` sudo apt install sox ```
+
+A list of distributions where the package is available can be found at: https://pkgs.org/download/sox
+
+#### 0.1 Download Deepspeech 
+
+Download a stable(!) release from the release page and extract it to a folder of your choice.
```

Comment:
Is `stable release` a model here ? Because `requirements.txt` already fits properly